src/ConfigWidgets/SymbolsDialog.cpp

```diff
@@ -85,12 +149,15 @@ ErrorMessageOr<void> SymbolsDialog::TryAddSymbolPath(const std::filesystem::path
   return outcome::success();
 }
 
-[[nodiscard]] std::vector<std::filesystem::path> SymbolsDialog::GetSymbolPaths() {
+[[nodiscard]] std::vector<std::filesystem::path> SymbolsDialog::GetSymbolPathsFromListWidget() {
   std::vector<std::filesystem::path> result;
-  result.reserve(ui_->listWidget->count());
 
   for (int i = 0; i < ui_->listWidget->count(); ++i) {
-    result.emplace_back(ui_->listWidget->item(i)->text().toStdString());
+    QListWidgetItem* item = ui_->listWidget->item(i);
+    ORBIT_CHECK(item != nullptr);
+    if (dynamic_cast<MappingItem*>(item) != nullptr) continue;
```

Comment:
Can you make this if statement if(item->type() != UserType) like you do below? It feels like that would be more consistent.

Reply:
Great idea! done